src/widgets/menu_bar.py

```diff
@@ -50,14 +51,20 @@ def widget_menu_creation(self):
     # Open command for option in menu
     def open_file(self):
         file_name = filedialog.askopenfilename(
-            title="Select .fits file",
-            filetypes=(("Fits files", "*.fits"), ("All files", "*.*")),
+            title="Select file",
+            filetypes=(
+                ("Fits/Png files", "*.fits *.png"),
```

Comment:
Imo this is definitely better than having two separate options for file type

Reply:
So keep or leave

Reply:
Definitely keep yeah was just sayin it's good